README.md

```diff
@@ -0,0 +1,35 @@
+# MXConnect Demo iOS App
+This is a simple application that shows how to get started with embedding an Connect into an iOS application
+
+
+## Quick Start
```

Comment:
I've always disliked "Quick Start" as a section heading, mostly because quick is subjective and often not true. I'd go with this:

```suggestion
## How to download and use the demo app
```